prometheus/doc/alert-manager.md

```diff
@@ -0,0 +1,53 @@
+OpenPai leverage [Prometheus Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to do
+alert. Alertmanager can send administrator an email to indicate cluster failure or problem with
+pre-configured rules.
+
+# Configuration
+
+To enable alert manager, you should configure it correctly:
+* select a node to deploy alert manager, it can be a master node of kubernetes or any other node. Once you selected, you should mark this node with `alert-manager: "true"` in `cluster-configuration` file, as [example](../../cluster-configuration/cluster-configuration.yaml)
+* configure alert manager by adding `alerting` fields under `prometheus` to services-configuration file. See [example configuration](../../cluster-configuration/services-configuration.yaml).
+
+`alerting` fields has following subfield:
+
+| Field Name | Description |
+| --- | --- |
+| alert_manager_port | port for alert manager to listen, make sure this port is not used in node |
+| alert_receiver | which email should receive alert email |
+| smtp_url | smtp server url for alert manager to connect |
+| smtp_from | this email address is where alerting email sent from |
+| smtp_auth_username | use this user name to login to smtp server. This user should be able to send email as `smtp_from`, can be same with `smtp_from` |
+| smtp_auth_password | use this password to login to smtp server |
+
+More advanced configurations for alert manager is not supported in pai, see official alert manager
+[document](https://prometheus.io/docs/alerting/configuration/) for more options.
+
+# Alerting rule
+
```

Comment:
To facilitate the OpenPAI usage, we had predefined few alerting rules for OpenPAI.